Samples~/ThirdPersonDemoURP/Bots/NewRGBot_n1024122209000000000/BotEntryPoint.cs

```diff
@@ -0,0 +1,84 @@
+using RegressionGames;
+using RegressionGames.RGBotLocalRuntime;
+using RGThirdPersonDemo;
+using UnityEngine;
+using Random = System.Random;
+
+namespace NewRGBot_n1024122209000000000
```

Comment:
This looks identical to `QuickstartBot`. Did you mean to remove this one?

Reply:
Fixed/Removed